src/ensembl/production/metadata/api/sample/ensembl_genome_metadata/dataset.txt

```diff
@@ -310,7 +310,7 @@
 356	90ba6c03-5161-4f9a-911c-1961b9c0470d	genebuild	ENS01	2023-09-22 15:06:46.000000	GCA_018472825.1_ENS01	188	2	Submitted
 357	4519fdf3-8b4e-463b-9822-69c45ee408da	assembly	\N	2023-09-22 15:06:46.000000	GCA_000001735.1	190	1	Submitted
 359	de92123a-22ca-407f-9954-d8c0f8b17f64	assembly	\N	2023-09-22 15:06:48.000000	GCA_018503575.1	192	1	Submitted
-361	e95e194c-52ad-4b1e-94d4-1d5c0a03e9e3	assembly	\N	2023-09-22 15:06:48.000000	GCA_018466835.1	193	1	Submitted
+361	e95e194c-52adc-4b1e-94d4-1d5c0a03e9e3	assembly	\N	2023-09-22 15:06:48.000000	GCA_018466835.1	193	1	Submitted
```

Comment:
Is it a typo? normally only 4 chars in between

Reply:
Yes it is.